docker-setups/angular-docker.md

```diff
@@ -205,7 +205,9 @@ services:
     image: angular-sample-image
```

Comment:
The service, image, APP_NAME, container and volume names should be consistent. So if we have the project name as angular-docker, then we should keep angular-docker for the first 4 and for volume, we should keep as angular-docker-db-data.

Reply:
Made the changes in the document as per the above instruction.